projects/ngx-sub-form/src/lib/ngx-sub-form.types.ts

```diff
@@ -8,15 +8,14 @@ export interface OnFormUpdate<FormInterface> {
   onFormUpdate?: (formUpdate: FormUpdate<FormInterface>) => void;
 }
 
-type Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
 type Nullable<T> = T | null;
 
 export type NullableObject<T> = { [P in keyof T]: Nullable<T[P]> };
 
-export type TypedFormGroup<FormInterface> = Omit<FormGroup, 'controls' | 'value'> & {
+export interface TypedFormGroup<FormInterface> extends FormGroup {
```

Comment:
I think it might be worth writing a comment explaining why we're not using Omit but up to you :+1:

Reply:
I kinda don't think so tbh - extending with interface is imo the more natural/simpler outcome. Plus there is a test which will not compile if anyone feels like they want to change it back.